core/domain/html_cleaner.py

```diff
@@ -25,8 +25,8 @@
 
 
 def filter_a(name, value):
-    """Returns whether the given attribute of an anchor ('a') tag should be 
-    whitelisted.
+    """Returns whether the given attribute of an anchor ('a') tag should be whit
```

Comment:
Er -- to be clear, I don't mean breaking off words at ends of lines either. No other docstring does this. I mean the following:

    """Returns whether the given attribute of an anchor ('a') tag should be
    whitelisted.


    """Cleans a piece of user submitted HTML. This only allows HTML from a
    restricted set of tags, attrs and styles.

etc. I.e., go as long as you can, until doing so will force you to break a word.